PubSub/tests/System/ManageSubscriptionsTest.php

```diff
@@ -411,7 +476,13 @@ private function assertSubsFound($class, $expectedSubs)
                 $sName = end($nameParts);
                 foreach ($expectedSubs as $key => $expectedSub) {
                     if ($sName === $expectedSub) {
-                        $foundSubs[$key] = $sName;
+                        if ($bucket) {
```

Comment:
Since we're not asserting the values of the storage config, wouldn't it be more intuitive to pass a flag variable(let's say `assertForStorageConfig` with the default value as false. 

Wdyt?